### ISSUE TITLE
Log In

### ISSUE THREAD

**Issue by Gilbert54** [ppawel/openstreetmap-website#28]
Log In with OSM account does not work.
Creating a new account does not work either: when I create one, the confirmation e-mail does not arrive (at least not within 30 minutes) -- Correction. I got the e-mail after about 1 hour. It does accept the newly created account, but it would still be nice if I could just use my OSM account ?

**Comment by ppawel** [comment]
http://owl.apis.dev.openstreetmap.org/ is running on the test server and it is not connected to the main user database. As far as I know it is not possible to connect a test instance to the main database because of security.

Registering new users should work (I will ask the admin about e-mail sending) but note that it is all on a test database.

**Comment by Gilbert54** [comment]
I was able to  log in with a new account on that server. But I suppose that
ultimately the goal is to use the same account on the regular OSM page as
on OWL - or maybe OWL just integrates with the OSM server, right?

Thanks for responding
Gilbert

**Comment by ppawel** [comment]
Yes, the goal is to integrate the history tab with the main website and users will be able to use their existing accounts.

**Comment by matkoniecz** [comment]
I would suggest to drop "log in" and "register", I was getting "wait, it looks like phishing attempt" on every noticing this part of menu before I removed this part of website with adblock.
